passkeeper/git.py

```diff
@@ -27,6 +27,11 @@ def add(self, files):
         for file in files:
             self._run_git_cmd('add %s' % file)
 
+    def clean(self):
+        self.init()
+        self._run_git_cmd('add -A')
```

Comment:
After call `init` you have a encrypted directory with file untrackable in your new git.

I think a simple git add encrypted will do the job (https://github.com/shaftmx/passkeeper/blob/master/passkeeper/git.py#L27)